ovh/data_dbaas_logs_test.go

```diff
@@ -0,0 +1,47 @@
+package ovh
+
+import (
+	"fmt"
+	"os"
+	"testing"
+
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/resource"
+)
+
+const testAccDataSourceDbaasLogs_dedicated = `
+data "ovh_dbaas_logs" "ldp" {
+ service_name  = "%s"
+}
+`
+
+func TestAccDataSourceDbaasLogs_dedicated(t *testing.T) {
+	serviceName := os.Getenv("OVH_DBAAS_LOGS_SERVICE_TEST")
+
+	config := fmt.Sprintf(
+		testAccDataSourceDbaasLogs_dedicated,
+		serviceName,
+	)
+
+	resource.Test(t, resource.TestCase{
+		PreCheck: func() { testAccPreCheckDbaasLogs(t) },
+
+		Providers: testAccProviders,
+		Steps: []resource.TestStep{
+			{
+				Config: config,
+				Check: resource.ComposeTestCheckFunc(
+					resource.TestCheckResourceAttr(
```

Comment:
The cluster_type is a `computed` attribute so you can't know in advance this attribute :)

Reply:
As tests run against a existing, known LDP instance, theses values are known in advance. We just have to agree on which one to use.